From 58e96fb8e4060ec48ca9fbb188458db64637be77 Mon Sep 17 00:00:00 2001
From: Steve deRosier <Steve.Derosier@lairdtech.com>
Date: Mon, 8 Jul 2013 15:50:56 -0700
Subject: [PATCH 2/4] Adjust the NAND OOB area for WB45

This changes the NAND OOB area so it's forward in the OOB and increases
the size used.
---
 drivers/mtd/nand/atmel_nand.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/mtd/nand/atmel_nand.c b/drivers/mtd/nand/atmel_nand.c
index 994dd9f..0b240db 100644
--- a/drivers/mtd/nand/atmel_nand.c
+++ b/drivers/mtd/nand/atmel_nand.c
@@ -720,10 +720,10 @@ static int atmel_pmecc_nand_init_params(struct nand_chip *nand,
  * several NAND_CMD_RNDOUT during read
  */
 static struct nand_ecclayout atmel_oobinfo_large = {
-	.eccbytes = 4,
-	.eccpos = {60, 61, 62, 63},
+	.eccbytes = 28,
+	.eccpos = {2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29},
 	.oobfree = {
-		{2, 58}
+		{30, 34}
 	},
 };
 
-- 
1.7.9.5

